core/tests/build_sources/assets/constants.js

```diff
@@ -30,6 +30,12 @@ var constants = {
   "DEFAULT_THUMBNAIL_ICON": "Lightbulb",
   "DEFAULT_CATEGORY_ICON": "Lightbulb",
 
+  // These categories are shown in the library navbar.
```

Comment:
Add a comment about the fact that they should be sorted.

Reply:
Done.